parsely/src/main/java/com/parsely/parselyandroid/LocalStorageRepository.kt

```diff
@@ -33,6 +40,10 @@ internal open class LocalStorageRepository(private val context: Context) {
         persistObject(ArrayList<Map<String, Any>>())
     }
 
+    fun remove(toRemove: List<Map<String, Any>>) {
+        persistObject(getStoredQueue() - toRemove.toSet())
+    }
```

Comment:
This method is not used in this PR. It'll be in next PRs, I just left it as I was pivoting from different ideas for scope of this PR.

Reply:
Good to know, thanks for clarifying that for me @wzieba ! 🙇

A follow-up question I had was related to the `purgeStoredQueue()` related test that got (kind of) removed, as it was placed with the `remove(...)` test. Will `purgeStoredQueue()` be deleting in next PRs maybe, if not I suggest keeping that test? 🤔

Reply:
> Will purgeStoredQueue() be deleting in next PRs maybe,

Yes, exactly - I'd prefer to have a granular control of what's removed from the local storage.